pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -122,6 +122,9 @@ public BaseBrokerRequestHandler(Configuration config, RoutingTable routingTable,
     } else {
       _tableCache = null;
     }
+
+    _enableBrokerQueryLimitOverride = _config.getBoolean(CommonConstants.Broker.CONFIG_OF_ENABLE_QUERY_LIMIT_OVERRIDE, false);
```

Comment:
(nit) `CommonConstants.Broker` is already imported so you can directly use `Broker.CONFIG_OF_ENABLE_QUERY_LIMIT_OVERRIDE`